lms/djangoapps/student_account/views.py

```diff
@@ -170,72 +163,43 @@ def _third_party_auth_context(request):
     """
     context = {
         "currentProvider": None,
-        "providers": []
+        "providers": [],
+        "finishAuthUrl": None,
+        "errorMessage": None,
     }
 
-    course_id = request.GET.get("course_id")
-    email_opt_in = request.GET.get('email_opt_in')
-    redirect_to = request.GET.get("next")
-
-    # Check if the user is trying to enroll in a course
-    # that they don't have access to based on country
-    # access rules.
```

Comment:
@bradenmacdonald looks like we should be in the clear here. The enrollment API is ultimately responsible for enrolling the user, and it includes [enforcement of country access rules](https://github.com/edx/edx-platform/blob/master/common/djangoapps/enrollment/views.py#L412-L423).